packages/react/src/components/Layer/Layer.base.tsx

```diff
@@ -95,7 +92,13 @@ export const LayerBase: React.FunctionComponent<ILayerProps> = React.forwardRef<
 
     // If a doc or host exists, it will remove and update layer parentNodes.
     const createLayerElement = () => {
-      const host = getHost();
+      const doc = getDocument(rootRef.current);
```

Comment:
Will `rootRef.current` ever be null? If so you will nullref.

Reply:
`createLayerElement` is only called in a `useLayoutEffect` hook, so I don't think `rootRef` can be undefined when rendered in a browser.